mri/reconstructors/self_calibrating.py

```diff
@@ -137,7 +142,7 @@ def __init__(self, fourier_op, linear_op=None, prox_op=None, mu=0,
         self.n_jobs = n_jobs
 
     def reconstruct(self, kspace_data, optimization_alg='pogm', x_init=None,
-                    num_iterations=100, reinit_grad_op=False, **kwargs):
+                    num_iterations=100, reinit_grad_op=True, **kwargs):
```

Comment:
Really `reinit_gradding_op` is really misconfusing, `recompute_smaps` is more informative since you changing the forward model therefore you recomputing the Lipschitz constant

Reply:
Changing it